### ISSUE TITLE
Incorrect game detection causing extraction failures

### ISSUE THREAD

**Issue by golyalpha** [gildor2/UModel#65]
As mentioned in issue #6, UModel with default settings (auto detect on compression and game) fails to extract/export any model and animation from the game files with an error of `unknown compression flags: 514`.

But unlike suggested, the game files are not encrypted/packed with an unknown packer, the problem is instead incorrect game detection.

With "Override game detection" enabled, UE3 engine and SMITE (another game by HiRez) selected, any scans/views/exports are successful (as long as there's something to extract of course).

TL;DR: There's a bug in the game detection system, causing decompression to fail (probably due to incorrect decompression setup for the particular game)

**Comment by gildor2** [comment]
This game was not added to autodetection by intention. The code structured in a way which is not allowing to retrieve game name in decompression code, so "force game" option is required. The same thing is true for several games, not just Smite or Paladins.

**Comment by gildor2** [comment]
In addition to what I said before. I'm not adding autodetection for online games. They're updating quite often, with changing package version numbers, so autodetection is broken too often. So I prefer users to tell to UModel which game they're opening. I think this is not an issue, but rather "how to use UModel with some game".
